omod/src/main/java/org/openmrs/module/queue/web/resources/QueueMetricsResource.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.queue.web.resources;
+
+import java.util.Arrays;
+
+import org.openmrs.Concept;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.queue.api.QueueService;
+import org.openmrs.module.queue.web.resources.custom.response.GenericSingleObjectResult;
+import org.openmrs.module.queue.web.resources.custom.response.PropValue;
+import org.openmrs.module.queue.web.resources.custom.response.QueueMetric;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.resource.impl.EmptySearchResult;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+
+@Resource(name = RestConstants.VERSION_1 + "/queue-metrics", supportedClass = QueueMetric.class, supportedOpenmrsVersions = {
+        "2.0 - 2.*" })
```

Comment:
Do you intentionally plan not to support version 3.0 of the platform? :)

Reply:
Not necessarily, most of the queue resources at the moment have been configured for platform "2.0 - 2.*"  I believe we should remove this restriction since we'll get to 3.* soon. I could do that on a separate PR